ponyo/simulate_expression_data.py

```diff
@@ -668,6 +668,11 @@ def shift_template_experiment(
     loaded_model.load_weights(weights_encoder_filename)
     loaded_decode_model.load_weights(weights_decoder_filename)
 
+    # Read data
+    normalized_data = normalized_data = pd.read_csv(
```

Comment:
Two `normalized_data` assignment.

Reply:
nice catch. I'm not sure why python doesn't throw an error for something like that...

Reply:
The syntax is correct (but obviously redundant), because Python allows you to do assignment like this:
```
a = b = 1
```
so Python silently assigned the same variable twice without any complaints.